### PR TITLE
NI-DCPower: Enable perform lcr load compensation

### PR DESCRIPTION
**Pull request by olsl21**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

- Enable "perform_lcr_load_compensation" function codegen
- Create system test `test_perform_lcr_load_compensation()` for it
- Update CHANGELOG

### What testing has been done?

Local build was successful. The newly added system test passed.

### Additional Info

This PR is currently dependent on #1771 for its metadata update.

### CODE REVIEW

File: generated/nidcpower/nidcpower/session.py [hunk 1]
```diff
@@ -5703,6 +5703,61 @@ def _parse_channel_count(self):
...
+
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | frequency            | The spot frequency.                                                                                                                                   |
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | reference_value_type | A known specification value of your DUT to use as the basis for load compensation. Refer to the enums.LCRReferenceValueType for the supported values. |
+                +----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | reference_value_A    | A value that describes the **reference_value_type** specification. Use as indicated by the **reference_value_type** option you choose.                |
```
Comment (olsl21) [inline generated/nidcpower/nidcpower/session.py:5747]: This docstring will be fixed after including the metadata update from #1771

Comment (sbethur) [inline generated/nidcpower/nidcpower/session.py:5747]: There is no ref value A and B in the public API. I'd suggest making this piece of documentation be identical to the docstring in LCRLoadCompensationSpot constructor.

**Review by sbethur — APPROVED** [review]

File: generated/nidcpower/nidcpower/session.py [hunk 2]
```diff
@@ -5703,6 +5703,61 @@ def _parse_channel_count(self):
...
+        Generates load compensation data for LCR measurements for the test spots you specify.
+
+        You must physically configure your LCR circuit with an appropriate reference load to use this method to generate valid load compensation data.
+
+        When you call this method:
+
+        -  The load compensation data is written to the onboard storage of the instrument. Onboard storage can contain only the most recent set of data.
+        -  Most NI-DCPower properties in the session are reset to their default values. Rewrite the values of any properties you want to maintain.
+
+        To apply the load compensation data you generate with this method to your LCR measurements, set the lcr_load_compensation_enabled property to True.
```
[the remark this answers is not in the document]
Comment (olsl21) [reply generated/nidcpower/nidcpower/session.py:5719]: This statement is related to "applying" the LCR load compensation data while this method is used to "perform" the LCR load compensation to generate the compensation data. Hence, the general workflow would be to "perform" the LCR load compensation using this method, then set the property along with other desired sourcing and measurement settings to acquire a compensated LCR measurement. The detailed documentation for LCR compensation is available at https://www.ni.com/docs/en-US/bundle/ni-dcpower-compensating-lcr-measurements/page/ni-dcpower-compensating-lcr-measurements.html.

Also, this docstring is consistent with the one in LabVIEW (https://www.ni.com/docs/en-US/bundle/ni-dcpower-21.0-labview-api-ref/page/nidcpowerviref/nidcpower_perform_lcr_load_compensation.html)
image, so I think it should be good enough for the users.

File: generated/nidcpower/nidcpower/session.py (hunk 1 above)
Comment (olsl21) [reply generated/nidcpower/nidcpower/session.py:5747]: As mentioned in https://github.com/ni/nimi-python/pull/1772/files/6dec3e3077ca16f01cb4ba7b856d40cb47f1759c#r924209556, this docstring will be fixed after including the metadata update from #1771

Comment (olsl21) [reply generated/nidcpower/nidcpower/session.py]: This docstring has been fixed in the latest merge commit.

Comment (olsl21) [reply generated/nidcpower/nidcpower/session.py]: This docstring has been fixed in the latest merge commit.

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**